src/Schema/Events/ImpressionTracking.ts

```diff
@@ -104,6 +104,28 @@ export interface ProgressiveOnboardingTooltipViewed {
   type: string
 }
 
+/**
+ * A user viewed a toast. `subject` will describe what toast it is.
+ *
+ * This schema describes events sent to Segment from [[viewedToast]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "viewedToast",
+ *    context_module: "artworkRecommendations",
+ *    context_screen_owner_type: "artworkGrid",
```

Comment:
These example values look swapped, right? The screen owner type is `artworkRecommendation` while the intended context module is `artworkGrid`?